### PR TITLE
Enable color by default only if os.Stdout is a TTY

### PR DESCRIPTION
**Pull request by ribasushi**

### CODE REVIEW

File: cli/util/color.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package cliutil
+
+import (
+	"os"
+
+	"github.com/mattn/go-isatty"
+)
+
+var DefaultColorUse = isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd())
```
Comment (magik6k) [inline cli/util/color.go:9]: Can we also have an env var for this? (`LOTUS_COLOR=1`, or sth like that, unless there is already a standard env var for that kind of thing)

Just `isatty.IsTerminal` will return false when running stuff in `watch --color ..` for example.

Comment (ribasushi) [reply cli/util/color.go:9]: So... we could introduce `LOTUS_COLOR=1` *OR* we could recognize `GOLOG_LOG_FMT=color` which is [a thing from `ipfs/go-log`](https://github.com/ipfs/go-log/pull/116#issuecomment-873779808)

I can go either way, just suggesting an alternative for smaller amount of options to keep track of...

@magik6k say the word

Comment (ribasushi) [reply cli/util/color.go:9]: @Stebalien please be weighing in as well, I surely missed other stuff here too ;)

Comment (magik6k) [reply cli/util/color.go:9]: `GOLOG_LOG_FMT` works for me

Comment (ribasushi) [reply cli/util/color.go]: @magik6k please be checking https://github.com/filecoin-project/lotus/pull/6696/commits/4dd59b566ced

**Review by magik6k — APPROVED** [review]

**Pull request merged**